src/OrchardCore/OrchardCore.Email.Abstractions/MailMessageBody.cs

```diff
@@ -0,0 +1,17 @@
+namespace OrchardCore.Email;
+
+/// <summary>
+/// Represents a body for <see cref="MailMessage"/>.
+/// </summary>
+public class MailMessageBody
+{
+    /// <summary>
+    /// Gets or sets the body in plain text format.
+    /// </summary>
+    public string Text { get; set; }
```

Comment:
```suggestion
    public string PlainText { get; set; }
```

Reply:
I used `Text` something similar to `BodyBuilder.Text`

Reply:
I just think that in contrast to "HTML" it's clearer to have "plain text". It's also a term commonly used for this when talking about e-mail.

Reply:
No problem we can rename it